json-guides/microprofile-config.json

```diff
@@ -158,7 +158,7 @@
                 "You can specify a value in the runtime environment for mandatory configuration properties. Properties are mandatory when the annotation does not specify an optional default value. When the mandatory configuration property value is not found in any of the ConfigSources, a <code>NoSuchElementException</code> is thrown during application startup."
             ],
             "instruction": [
-                "To inject a mandatory <code>port</code> configuration property into the code, add the following <code>@Inject</code> and <code>@ConfigProperty</code> annotations to line 9 before declaring the <code>private int port</code>.<br><codeblock>@Inject @ConfigProperty(name=\"port\")</codeblock><br>Alternatively, click  <action title='Inject a ConfigProperty' onclick='microprofileConfigCallBack.addInjectConfigButton(event, \"ConfigureViaInject\")'>Add</action>.<br>Then, click <action title='Run' onclick=\"microprofileConfigCallBack.saveTabbedEditorButton(event, 'ConfigureViaInject')\">Run</action> on the editor menu pane to see the exception occur."
+                "To inject a mandatory <code>port</code> configuration property into the code, add the following <code>@Inject</code> and <code>@ConfigProperty</code> annotations to line 9 before declaring the <code>private int port</code>.<br><codeblock>@Inject @ConfigProperty(name=\"port\")</codeblock><br>Alternatively, click  <action title='Add code to inject a ConfigProperty' onclick='microprofileConfigCallBack.addInjectConfigButton(event, \"ConfigureViaInject\")'>Add</action>.<br>Then, click <action title='Run' onclick=\"microprofileConfigCallBack.saveTabbedEditorButton(event, 'ConfigureViaInject')\">Run</action> on the editor menu pane to see the exception occur."
```

Comment:
missing on onkeypress()

Reply:
This is a DAP issue and not relevant to the subject of this issue. It should be addressed in #254.